courses/models_test.py

```diff
@@ -154,27 +154,29 @@ def test_present_enroll_closed_with_future_present(self):
             enr_start=self.from_weeks(40),
             enr_end=self.from_weeks(50),
         )
-        next_run = self.course.get_first_unexpired_run()
+        next_run = CourseRun.get_first_unexpired_run(self.course)
         assert isinstance(next_run, CourseRun)
         assert next_run.pk == course_run.pk
 
     def test_exclude_course_run(self):
         """
-        Two runs, one of which will be excluded from the results.
+        Two runs, one of which will be excluded from the results because not upgradable
```

Comment:
Maybe I just don't understand this part of the codebase, but I don't know what you mean by "upgradable". Can you clarify?